apps/frontend/src/components/technique/TechniqueTable.tsx

```diff
@@ -0,0 +1,256 @@
+import { Column } from '@material-table/core';
+import { Dialog, DialogContent, Typography } from '@mui/material';
+import i18n from 'i18n';
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useQueryParams } from 'use-query-params';
+
+import { useCheckAccess } from 'components/common/Can';
+import ScienceIcon from 'components/common/icons/ScienceIcon';
+import SuperMaterialTable, {
+  DefaultQueryParams,
+  UrlQueryParamsType,
+} from 'components/common/SuperMaterialTable';
+import { useTechniquesData } from 'hooks/technique/useTechniquesData';
+import useDataApiWithFeedback from 'utils/useDataApiWithFeedback';
+import { FunctionType } from 'utils/utilTypes';
+
+import AssignedInstrumentsTable from './AssignedInstrumentsTable';
+import AssignInstrumentsToTechniques from './AssignInstrumentsToTechniques';
+import CreateUpdateTechnique from './CreateUpdateTechnique';
+import {
+  InstrumentFragment,
+  TechniqueFragment,
+  UserRole,
+} from '../../generated/sdk';
+
+const TechniqueTable = () => {
+  const {
+    loadingTechniques,
+    techniques,
+    setTechniquesWithLoading: setTechniques,
+  } = useTechniquesData();
+
+  const { api } = useDataApiWithFeedback();
+  const isUserOfficer = useCheckAccess([UserRole.USER_OFFICER]);
+  const [urlQueryParams, setUrlQueryParams] =
+    useQueryParams<UrlQueryParamsType>(DefaultQueryParams);
+
+  const [openTechniqueAssignment, setOpenTechniqueAssignment] = useState(false);
+  const [selectedTechnique, setSelectedTechnique] =
+    useState<TechniqueFragment | null>(null);
+
+  const { t } = useTranslation();
+
+  const columns: Column<TechniqueFragment>[] = [
+    {
+      title: 'Name',
+      field: 'name',
+    },
+    {
+      title: 'Short code',
+      field: 'shortCode',
+    },
+    {
+      title: 'Description',
+      field: 'description',
+    },
+    {
+      title: i18n.format(t('instrument'), 'plural'),
+      render: (data) => data.instruments.length,
+    },
+  ];
+
+  const onTechniqueDelete = async (techniqueDeletedId: number | string) => {
+    try {
+      await api({
+        toastSuccessMessage: t('Technique') + ' deleted successfully!',
+      }).deleteTechnique({
+        id: techniqueDeletedId as number,
+      });
+
+      return true;
+    } catch (error) {
+      return false;
+    }
+  };
+
+  const AssignmentIndIcon = (): JSX.Element => <ScienceIcon />;
```

Comment:
Can we use the Icon, that we use for assigning the Instrument to the Proposal or Call. 
<img width="1496" alt="Screenshot 2024-07-01 at 11 29 01" src="https://github.com/UserOfficeProject/user-office-core/assets/132274772/38f5ea03-c516-40df-aca4-ec25ac349541">

Available in `user-office-core/apps/frontend/src/components/common/icons/ScienceIcon.tsx`

Reply:
Hi Yoganandan, I think we may have already changed this in https://github.com/UserOfficeProject/user-office-core/pull/628/commits/24be98e51c4797b7d0bf5583d1d5b280e23c9f9d, if I've understood you correctly.



This is what it looks like now:

![image](https://github.com/UserOfficeProject/user-office-core/assets/14293717/4cd29d17-3b6e-46a4-97c1-6de16047a82f)

Reply:
@simonfernandes Oh That's my bad. I checked the Video and it had the different Icon. But, I should have checked the code completely.